test/DisintermediatedGrants.test.js

```diff
@@ -286,85 +287,97 @@ describe("DisintermediatedGrants", function () {
         })
     })
     describe("grant proposals", function () {
-        it("cannot be created by non-owner", async function () {
+        it("cannot be created by non-multisig", async function () {
             const donationId = await setDonation(this.grants, this.defaultNativeDonation)
             await expect(
-                this.grants.connect(this.eve).proposeGrant(donationId, this.eve.address, ETH_AMOUNT)
-            ).to.be.revertedWith("Ownable: caller is not the owner")
+                this.grants.connect(this.eve).proposeGrant({
+                    donationId,
+                    recipient: this.eve.address,
+                    amount: ETH_AMOUNT,
+                })
+            ).to.be.revertedWith("caller is not the multisig")
         })
         it("fail if donation does not exist", async function () {
-            await expect(this.grants.proposeGrant(404, this.eve.address, ETH_AMOUNT.mul(2))).to.be.revertedWith(
-                "donation cannot cover full grant amount"
-            )
+            await expect(
+                this.grants.connect(this.multisig).proposeGrant({
+                    donationId: 404,
+                    recipient: this.eve.address,
+                    amount: ETH_AMOUNT.mul(2),
+                })
+            ).to.be.revertedWith("donation cannot cover full grant amount")
```

Comment:
Bad error message

Reply:
Fixed in [github.com/researchdao/contracts/commit/377339512f875c3b18da257ac79e96f12fee3e76](https://github.com/researchdao/contracts/commit/377339512f875c3b18da257ac79e96f12fee3e76).